### PR TITLE
fix: panic on precommits does not have any +2/3 votes

### PR DESCRIPTION
**Pull request by lklimek**
## Issue being fixed or feature implemented

```
BUG: CONSENSUS FAILURE!!! - entering precommit wait step (12928/0), but precommits does not have any +2/3 votes
```

## What was done?

Some parts of the code were not correctly modified to support LLMQ quorums.
This PR fixes this behavior.

## How Has This Been Tested?

Added unit test, to be tested on schnapps once we have a pre-release

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [ ] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

**Review by shumkov — APPROVED** [review]
Good catch! 👍

File: types/vote_set_test.go [hunk 1]
```diff
@@ -519,6 +523,112 @@ func TestVoteSet_MakeCommit(t *testing.T) {
...
+			llmqType:      btcjson.LLMQType_50_60,
+			numValidators: 50,
+			threshold:     30,
+		},
+	}
+
+	for ti, tt := range testCases {
+		name := strconv.Itoa(ti)
+		t.Run(name, func(t *testing.T) {
+			height, round := int64(1), int32(0)
```
Comment (shotonoff) [inline types/vote_set_test.go:557]: what if to declare `height` and `round` out of this function? or to move its inside `randVoteSetWithLLMQType`?

File: types/vote_set_test.go [hunk 2]
```diff
@@ -519,6 +523,112 @@ func TestVoteSet_MakeCommit(t *testing.T) {
...
+			numValidators := tt.numValidators
+			llmqType := tt.llmqType
+			threshold := tt.threshold
```
Comment (shotonoff) [inline types/vote_set_test.go:560]: I am not sure that is necessary having a copy of fields from `test-case` structure

**Review by shotonoff — APPROVED** [review]
left a couple of minor comments

**Review by QuantumExplorer — APPROVED** [review]

**Pull request merged**